modules/custom/cu_rss_block/cu_rss_block.module

```diff
@@ -191,23 +191,19 @@ function cu_rss_block_secure_permissions($role) {
       'edit any rss bean',
       'view any rss bean',
     ),
-
     'content_editor' => array(
```

Comment:
Maybe move these to a bundle while you are at it? I think all permissions should go into `express_permissions` or a bundle's module file.